apollo-runtime/src/main/java/com/apollographql/apollo/internal/interceptor/ApolloServerInterceptor.java

```diff
@@ -97,16 +97,20 @@ public void interceptAsync(@NotNull final InterceptorRequest request, @NotNull f
           }
 
         } catch (IOException e) {
-          logger.e(e, "Failed to prepare http call for operation %s", request.operation.name().name());
-          callBack.onFailure(new ApolloNetworkException("Failed to prepare http call", e));
+          final String operationName = request.operation.name().name();
+          final String message = String.format("Failed to prepare http call for operation [%s]", operationName);
```

Comment:
Nitpick as we're using single quotes in other places
```suggestion
          final String message = String.format("Failed to prepare http call for operation '%s'", operationName);
```